src/flux/app-config.coffee

```diff
@@ -0,0 +1,31 @@
+# Store for getting information about the application's configuration
+
+flux = require 'flux-react'
+
+{makeSimpleStore} = require './helpers'
+
+AppConfig =
+
+  setAssetsHost: (host) ->
+    @_assetsHost = host
+
+  exports:
+    getAssetsHost: ->
+      return @_assetsHost if @_assetsHost?
+      # http://caniuse.com/#feat=css-sel3 (substring selector is IE 8+)
+      tutorStyle = document.head.querySelector('link[href$="tutor.css"]')
```

Comment:
Ah, good to know.  I'll have to use the substring matcher then instead of the tail one.